magicblock-ledger/src/store/api.rs

```
@@ -327,6 +330,7 @@ impl Ledger {
         self.blockhash_cf.put(slot, &blockhash)?;
         self.blockhash_cf.try_increase_entry_counter(1);
         self.latest_block.store(slot, blockhash, timestamp);
+        let _ = self.block_txn_indexes.put(slot, AtomicU32::new(0));
```

are we just ignoring an error here?